### PR TITLE
v0.3.0

### PR DESCRIPTION
**Pull request by japaric**

### CODE REVIEW

**Comment by japaric** [comment]
@homunkulus r+

**Comment by homunkulus** [comment]
:pushpin: Commit def4fc8 has been approved by `japaric`

**Comment by homunkulus** [comment]
:hourglass: Testing commit def4fc8079dcb646ef3cab446a4b160e09e169bf with merge b73263e8fa534233e8c097c6b1e6e430ea160d13...

File: src/lib.rs [hunk 1]
```diff
@@ -37,18 +37,18 @@
...
-//! [`svd2rust`]: https://docs.rs/svd2rust/0..0/svd2rust/
+//! [`svd2rust`]: https://docs.rs/svd2rust/0.12.0/svd2rust/
 //! [`<cpu>`]: https://www.keil.com/pack/doc/CMSIS/SVD/html/elem_cpu.html
 //!
 //! # `app!`
 //!
 //! The `app!` macro is documented [here].
 //!
-//! [here]: https://docs.rs/cortex-m-rtfm-macros/0.2.0/cortex_m_rtfm_macros/fn.app.html
+//! [here]: https://docs.rs/cortex-m-rtfm-macros/0.2.1/cortex_m_rtfm_macros/fn.app.html
```
Comment (jonas-schievink) [inline src/lib.rs]: You bumped the version to `0.3.0` but this refers to `0.2.1`

**Comment by homunkulus** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/japaric/cortex-m-rtfm/builds/329247722?utm_source=github_status&utm_medium=notification)
Approved by: japaric
Pushing b73263e8fa534233e8c097c6b1e6e430ea160d13 to master...

File: Cargo.toml [hunk 2]
```diff
@@ -13,23 +13,21 @@ repository = "https://github.com/japaric/cortex-m-rtfm"
 version = "0.3.0"
 
 [dependencies]
-cortex-m = { git = "https://github.com/japaric/cortex-m" }
+cortex-m = "0.4.0"
+cortex-m-rtfm-macros = "0.3.0"
```
Comment (jonas-schievink) [inline Cargo.toml]: Is it not possible to keep the `path` dependency? I *think* Cargo will automatically rewrite it and publish both crates together.

Comment (japaric) [reply Cargo.toml]: I can try that next time.

File: src/lib.rs (hunk 1 above)
Comment (japaric) [reply src/lib.rs]: :-(

**Pull request merged**